app/models/account.rb

```diff
@@ -69,13 +69,13 @@ def self.columns
     [:buyer_accounts, { action: :destroy, class_name: 'Account' }],
     [:payment_gateway_setting, { action: :destroy, has_many: false }],
     [:profile, { action: :delete, has_many: false }],
-    [:templates, { action: :delete }],
+    [:templates, { action: :delete, class_name: 'CMS::Template' }],
```

Comment:
Good catch

Reply:
Gold! Thanks! :clap:

Reply:
I've just pushed it applying it to all models :+1: